src/fmeval/eval_algorithms/summarization_accuracy_semantic_robustness.py

```diff
@@ -56,6 +55,7 @@
     generate_mean_delta_score,
     generate_model_predict_response_for_dataset,
 )
+from fmeval.helper_models import BertscoreModelTypes
```

Comment:
Apologies if I missed it earlier.

We should be sticking to the terminology of HelperModels. This is gonna be useful in future too when we introduce other helper models.